nanodbc/nanodbc.h

```diff
@@ -157,6 +157,18 @@ typedef std::string string;
 #define NANODBC_TEXT(s) s
 #endif
 
+#ifdef NANODBC_USE_IODBC_WIDE_STRINGS
+typedef std::u32string wide_string;
+#else
+#ifdef _MSC_VER
+typedef std::wstring wide_string;
+#else
+typedef std::u16string wide_string;
+#endif
+#endif
+
+typedef wide_string::value_type wide_char_t;
```

Comment:
1. Any particular reason to not to rename to `nanodbc::wchar_t`  and `nanodbc::wstring`?

2. So, if built with `NANODBC_ENABLE_UNICODE` then `nanodbc::string` and `nanodbc::wide_string` alias the same type (eg. `std::wstring`), but if Unicode is not enabled, then they are distinct string types. Is that the intention?

Reply:
1. I would prefer not to use the standard names, as this could result in some strange errors when the nanodbc:: namespace is mistakenly omitted.
2. Yes thats basically it.